interfaces/acados_c/ocp_nlp_interface.c

```diff
@@ -501,6 +501,9 @@ void ocp_nlp_out_get(ocp_nlp_config *config, ocp_nlp_dims *dims, ocp_nlp_out *ou
 int ocp_nlp_dims_get(ocp_nlp_config *config, ocp_nlp_dims *dims, ocp_nlp_out *out,
```

Comment:
I don't get the meaning of this function.
A function called `ocp_nlp_dims_get` should take as fields `nx`, `nu`, `ny`, `nbx`, ... and not the name of model members...

Reply:
The function name should probably be changed given the confusion it creates. The point is that when you need to set/get something you can ask the core for its dimension in order to enforce consistency and avoid memory corruptions, It should really implement a map from attributes to their dimensions ("x" -> nx, "lbx" -> nbx and so on)

Reply:
see 90f0d9c